app/javascript/components/widgets/numbers/panel.jsx

```diff
@@ -1,6 +1,119 @@
 import React from 'react';
-import Wrapper from '../../panel-wrapper';
+import PropTypes from 'prop-types';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import styled from 'styled-components';
+import { over, lensPath, inc } from 'ramda';
+import pluralize from 'pluralize';
+import {
+  colors,
+  weights,
+  fontSizes,
+  spacing,
+  fonts,
+  leftPanelWidth,
+} from '../../../lib/theme';
 
-export const Numbers = () => <Wrapper />;
+const LoadingMessage = () => <p>Loading...</p>;
+const ErrorMessage = ({ message }) => <p>Error: {message}</p>;
+ErrorMessage.propTypes = {
+  message: PropTypes.string.isRequired,
+};
+
+const getEventCounts = gql`
+  {
+    events {
+      count {
+        githubPull
+        githubCommit
+        slackMessage
+      }
+    }
+  }
+`;
+
+const NumbersTitle = styled.div`
+  color: ${colors.white};
+  font-size: ${fontSizes.medium};
+`;
+
+const BigNumber = styled.div`
+  color: ${colors.white};
+  weight: ${weights.bold};
+  font-size: ${fontSizes.huge};
+`;
+
+const BigNumberSubtitle = styled.div`
+  font-size: ${fontSizes.medium};
+`;
+
+const NumberWrapper = styled.div`
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: space-between;
+  width: 5em;
+`;
+
+class SubscribedEvents extends React.PureComponent {
+  render() {
+    const { githubPull, githubCommit, slackMessage } = this.props;
+
+    const commitText = pluralize('Commit', githubCommit);
```

Comment:
The inconsistent case (titlecase vs. the following, which are lowercase) here bumped me when I first saw it. Not asking for a change, but food for thought:

```javascript
const commitText = pluralize('commit', githubCommit);
...
<BigNumberSubtitle style={{ color: colors.pink }}>
  {titlecase(commitText)} // not sure if this proj has a dep that can provide `titlecase`...
</BigNumberSubtitle>
```

or even:

```javascript
<BigNumberSubtitle style={{ color: colors.pink }}>
  {pluralize('Commit', githubCommit)}
</BigNumberSubtitle>
```

The code as-is looks very clean, so I'm not advocating for those changes (just something that crossed my mind).